PythonPonies/bangazon-storefront
# Swh tests

# Description
This pull request includes the following additions:
- Unit tests for PaymentType model + view
- PaymentType model
- PaymentType view
- Enables user to access their existing payment types and add new payment types to associate with their account. 

## Number of Fixes
0 fixes. 3 new files. 

## Related Ticket(s)
#3

## Problem to Solve
The first step in the execution of this product was to design the application by writing unit tests. In this pull request, I have addressed the writing of tests for the model for payment types along with the initial tests and code for the payment type view.

## Expected Behavior
- All tests that I have written should pass. 
- New table in database should be created (PaymentType)

## Steps to Test Solution

1. git fetch 
2. git checkout swh-tests
3. rm db.sqlite3
4. python manage.py makemigrations
5. python manage.py migrate
6. python manage.py test bangazon_storefront

- Ensure that tests pass (One test will fail - Ike's from the previous merge)
- Check the database out in db browser to ensure that new table has been created.


## Testing

[x] There are new unit tests in this PR, and I verify that there is full coverage of all new code.
[x] I certify that all existing tests pass

## Documentation
[x] There is new documentation in this pull request that must be reviewed..
[x] I added documentation for any new classes/methods

## Deploy Notes
N/A
